tests/plotting/test_plotting.py

```diff
@@ -2164,17 +2164,6 @@ def test_set_viewup():
     p.show()
 
 
-def test_plot_remove_scalar_bar(sphere):
-    sphere['z'] = sphere.points[:, 2]
-    plotter = pyvista.Plotter()
-    plotter.add_mesh(sphere, show_scalar_bar=False)
-    plotter.add_scalar_bar(interactive=True)
-    assert len(plotter.scalar_bars) == 1
-    plotter.remove_scalar_bar()
-    assert len(plotter.scalar_bars) == 0
-    plotter.show()
-
-
```

Comment:
This test largely overlapped with a test from `tests/test_scalar_bars.py` that I updated a bit.

But now that I'm writing this I realised that this test had added value via the regression test. Is it worth keeping? Or adding more tests here? If we scrap it then I also have to remove the corresponding image regression test.

On a related note, I was going to move `test/test_scalar_bars.py` to `test/plotting/`, but then I realised that this would need adding image regression testing (`pl.show()` calls for instance) and for those test functions it might not add much value. Perhaps we should do the move after all, and add more regression tests... I'm not really sure what to do with these.

Reply:
Hopefully, all of this is made easier with the image regression fixture? I'm +1 for moving `test/test_scalar_bars.py` to `tests/plotting/`

Reply:
I moved the relevant tests to plotting. Agreed that "seeing" is believing.